### PR TITLE
[Doc]Release notes for 7.8

### PR DESCRIPTION
**Pull request by karenzone**
This PR starts with the generated output from #11945, and builds on it. 

**Preview:** https://logstash_12033.docs-preview.app.elstc.co/guide/en/logstash/7.8/logstash-7-8-0.html

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -29,6 +30,170 @@ This section summarizes the changes in the following releases:
...
+===== Expanded JDK ecosystem 
+
+We have improved test scripts infrastructure to accept a specific Java
+Development Kit (JDK) present in the host. The changes include exporting the
+path to a JDK home into an environment variable named BUILD_JAVA_HOME and its
+use by shell scripts to launch test and integration scripts. This work and other
+JDK14 fixes pave the way for Logstash to support both AdoptOpenJDK 11 and 14 in
+the near future . https://github.com/elastic/logstash/pull/11786[#11786],
+https://github.com/elastic/logstash/pull/11839[#11839],
+https://github.com/elastic/logstash/pull/11935[#11935]
```
Comment (karenzone) [inline docs/static/releasenotes.asciidoc:47]: I'd like to rework this section to be more user-focused. Please help me understand the user benefits.

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -29,6 +30,170 @@ This section summarizes the changes in the following releases:
...
+Azure and Netflow modules in Logstash have been deprecated and replaced by
+the Azure modules in {filebeat-ref}/filebeat-module-azure.html[{filebeat}] and
+{metricbeat-ref}/metricbeat-module-azure.html[{metricbeat}], and the Netflow
+module in {filebeat-ref}/filebeat-module-netflow.html[{filebeat}]. The
+{filebeat} and {metricbeat} modules are compliant with the
+{ecs-ref}/index.html[Elastic Common Schema (ECS)].
+
+==== Known issue
+
+*Performance regression.* A potential performance regression may affects
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:105]:
```suggestion
*Performance regression.* A potential performance regression may affect
```

File: docs/static/releasenotes.asciidoc [hunk 3]
```diff
@@ -29,6 +30,170 @@ This section summarizes the changes in the following releases:
...
+some users. This issue can cause a slowdown on pipeline compilation when
+multiple pipelines are in use. We believe the issue was introduced in 7.7.0.
```
Comment (jsvd) [inline docs/static/releasenotes.asciidoc:107]:
```suggestion
multiple large pipelines are in use. We believe the issue was introduced in 7.7.0.
```

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (jsvd) [reply docs/static/releasenotes.asciidoc:47]: for now there isn't a user facing benefit, this is more to let users know we're expanding how logstash is tested, and that will create a comfortable baseline for us to be more flexible and reactive to supporting new JDK and deprecating old ones.

**Comment by karenzone** [comment]
Hi everybody. I merged this PR so that we aren't holding up the release.  If you have comments or fixes you'd like to see, I'll open a new PR to handle them.

**Comment by karenzone** [comment]
Opened #12037 to forwardport to 7.x branch.

File: docs/static/releasenotes.asciidoc (hunk 1 above)
Comment (roaksoax) [reply docs/static/releasenotes.asciidoc:47]: IMHO, we should have consolidated both "Expanded JDK Ecosystem" and "Expanded JDK Platform Support" and completely reword this as a user level benefit, further expanding that our testing infrastructure now covers this. Also, I think we should not longer link pull requests in this section, but link them on a further section below with the rest of the changelog.
